addons/account/models/account_journal_dashboard.py

```diff
@@ -423,7 +423,7 @@ def create_cash_statement(self):
     def action_open_reconcile(self):
         if self.type in ['bank', 'cash']:
             # Open reconciliation view for bank statements belonging to this journal
-            bank_stmt = self.env['account.bank.statement'].search([('journal_id', 'in', self.ids)]).mapped('line_ids')
+            bank_stmt = self.env['account.bank.statement'].search([('journal_id', 'in', self.ids)]).mapped('line_ids')[:1000]
```

Comment:
something is wrong. This way we'll always load first 1000 lines, not "next 1000 unreconcilled lines"

Reply:
`self.env['account.bank.statement.line'].search([('journal_id', 'in', self.ids), ('is_reconciled', '=', False))])` ?

Reply:
is_reconciled is in v14 only :cry: